### PR TITLE
Policy for user-installed software

### PR DESCRIPTION
**Pull request by willfurnass**

### CODE REVIEW

**Comment by willfurnass** [comment]
@gyengen Is there a suitable place this can be added to the HPC training material?

File: hpc/installing-software.rst [hunk 1]
```diff
@@ -21,9 +21,19 @@ This option should be seen as a service without support as we will expect such u
...
-.. raw:: html
+    :underline-bold:`Policy on user-installed software on University of Sheffield HPC systems`
 
-    <hr class="hr-mid-section-separator">
+    |br|
+
+    * Users should endeavour to download source code or software binaries 
+         * produced by trusted developers/vendors and
+         * acquired from trusted repositories/locations.
+    * User should remove any software that is definitely no longer needed.
```
Comment (jkwmoore) [inline hpc/installing-software.rst:31]:
```suggestion
    * Users should remove any software that is definitely no longer needed.
```

File: hpc/installing-software.rst [hunk 2]
```diff
@@ -21,9 +21,19 @@ This option should be seen as a service without support as we will expect such u
...
 
-.. raw:: html
+    :underline-bold:`Policy on user-installed software on University of Sheffield HPC systems`
 
-    <hr class="hr-mid-section-separator">
+    |br|
+
+    * Users should endeavour to download source code or software binaries 
+         * produced by trusted developers/vendors and
+         * acquired from trusted repositories/locations.
```
Comment (jkwmoore) [inline hpc/installing-software.rst:30]:
```suggestion
         * acquired from trusted repositories/locations.
    * Users should keep software up to date where reproducibility is not a concern.
```

**Review by nikishe — APPROVED** [review]
LGTM

**Review by Carldkennedy — APPROVED** [review]
LGTM

**Pull request merged**